### DISCUSSION TITLE
oauthPassThru support

### DISCUSSION THREAD

**Discussion by dastein1** [grafana/grafana-infinity-datasource#246] [Q&A]
With the recent backend rewrite it would be a trivial to support _oauthPassThru_. Though it has been commented out (probably on purpose). Would you accept a PR that enables that option and passes the authorization header from the client to the backend?

**Comment by yesoreyeram** [comment]
Sure. Happy to accept a PR. ( I have had purposefully commented out those because of not enough tests added to prove that it is working )
